Packs/AzureLogAnalytics/Integrations/AzureLogAnalytics/AzureLogAnalytics.yml

```diff
@@ -25,7 +25,6 @@ configuration:
   section: Connect
   display: Server URL
   advanced: true
-  defaultvalue: https://management.azure.com
```

Comment:
Have we checked how this change affect the instance?

Reply:
I added this default value by mistake in the original PR, and it shouldn't be there to begin with as it can lead to undesired endpoint setting.

Reply:
But did we check what happens if it is configured and then the version is being updated? do we handle it gracefully?

Reply:
In the current marketplace version, this _defaultvalue_ is an immediate bug if you have any configuration for an `Azure Cloud` parameter that is **not** the default `Worldwide` option.
In this PR, where _defaultvalue_ is removed, the affects will be as following:
1. If the customer uses "regular" worldwide credentials: It is transparent to them.
2. If the customer uses "GCC High" credentials: It fixes a problem for them where the resource_manager endpoint was set to `https://management.azure.com` by default regardless of the `Azure Cloud` parameter value.
This logic is due to the backward compatibility support in `MicrosoftApiModule`: https://github.com/demisto/content/blob/2c4f4b5cce0b8e772693e17faea6f0ba658fa2c6/Packs/ApiModules/Scripts/MicrosoftApiModule/MicrosoftApiModule.py#L581C47-L594C48